plugins/flytekit-spark/tests/test_wf.py

```diff
@@ -4,7 +4,7 @@
 
 import flytekit
 from flytekit import kwtypes, task, workflow
-from flytekit.types.schema import FlyteSchema
```

Comment:
Could we revert this?  It's used below.

Reply:
I don't think this is required in the latest version. LMK if otherwise.